app-header/demo/pesto.html

```diff
@@ -1,97 +0,0 @@
-<!--
```

Comment:
This demo's broken, and there's already the Pesto template, so I'm going to just remove it.